gslib/tests/test_mb.py

```diff
@@ -308,7 +303,7 @@ def test_create_with_custom_dual_regions_via_l_flag(self):
                                   suppress_consec_slashes=False)
     self.RunGsUtil(['mb', '-l', 'us-east1+us-east4', suri(bucket_uri)])
     stdout = self.RunGsUtil(['ls', '-Lb', suri(bucket_uri)], return_stdout=True)
-    self.assertRegex(stdout, r"locations:\t\t\[\'US-EAST1\', \'US-EAST4")
+    self.assertRegex(stdout, r"ocations:\s*\[\s*.US-EAST1.,\s*.US-EAST4")
```

Comment:
So I just realized that we have not added --format=gsutil to the ls command yet https://github.com/GoogleCloudPlatform/gsutil/blob/6fee45bb59c6124524d3930a5daca88a701ccd9c/gslib/commands/ls.py#L337 , because of which we need to make this adjustments. I

I  feel that if we just add `--format=gsutil` the formatting should be exactly like gsutil. Can you try that first?